### ISSUE TITLE
Feature - more share options

### ISSUE THREAD

**Issue by octoshrimpy** [Livinglist/Hacki#130]
It would be nice to share the link to HN, or to the article. 

For example, there was a post recently about https://eli5.gg. I want to share it with my wife, but she doesn't care about HN comments, just the product. having a `share HN` `share article` distinction would be awesome

**Comment by Livinglist** [comment]
Thx! That’s a good addition for next release.

**Comment by Fadyio** [comment]
I was about to say the same thing, this is a much-needed feature.
@Livinglist and I want to express my gratitude for the excellent work, Thank you.
